libgeopmd/src/geopm_sched.c

```diff
@@ -118,9 +118,9 @@ int geopm_sched_proc_cpuset_pid(int pid, int num_cpu, cpu_set_t *cpuset)
     uint32_t *proc_cpuset = NULL;
     FILE *fid = NULL;
 
-    char status_path[NAME_MAX];
-    int nprint = snprintf(status_path, NAME_MAX, "/proc/%d/status", pid);
-    if (nprint == NAME_MAX) {
+    char status_path[GEOPM_NAME_MAX];
+    int nprint = snprintf(status_path, GEOPM_NAME_MAX, "/proc/%d/status", pid);
```

Comment:
I think this one should have been PATH_MAX before since it's intended to be a multi-component path (though I guess incidentally it wouldn't ever exceed NAME_MAX anyways).